library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -81,6 +86,45 @@ data class Conversations(
         )
     }
 
+    fun newGroup(accountAddresses: List<String>): Group {
+        if (accountAddresses.isEmpty()) {
+            throw XMTPException("Cannot start an empty group chat.")
+        }
+        if (accountAddresses.size == 1 &&
+            accountAddresses.first().lowercase() == client.address.lowercase()
+        ) {
+            throw XMTPException("Recipient is sender")
+        }
+        val contacts = accountAddresses.map { client.contacts.find(it) }
```

Comment:
Is this only checking for the existence of a v2 bundle?

Not a blocker here, but we're going to have to update this here to check if the recipient has a v3 installation on the network. We probably need to expose this from libxmtp if we haven't already

Reply:
Good call. I think there is a canMessage function already I'll expose that and us that here instead.